### PR TITLE
[23.0] Media player fix issue 16415

### PR DESCRIPTION
**Pull request by bdwheele**
This fixes the media player's streaming issue mentioned in https://github.com/galaxyproject/galaxy/issues/16415

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [x] Instructions for manual testing are as follows:
  1. load an audio or video dataset 
  2. use the audio player or video player visualization

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: lib/galaxy/web/framework/base.py [hunk 1]
```diff
@@ -539,9 +539,9 @@ def send_file(start_response, trans, body):
         start = None
         end = None
         if trans.request.range:
-            start = trans.request.range.start
-            end = trans.request.range.end
-            file_size = trans.response.headers["content-length"]
+            start = int(trans.request.range.start)
+            file_size = int(trans.response.headers["content-length"])
+            end = int(file_size if end is None else trans.request.range.end)
```
Comment (mvdbeek) [inline lib/galaxy/web/framework/base.py:544]: I'm not sure this is quite correct, if `trans.request.range.end` is not sent or not parsed correctly we should probably respond with a 416. I'll take a look at what the media player is sending.

Comment (bdwheele) [reply lib/galaxy/web/framework/base.py:544]: When I originally encountered the error, request.range.end was None.  I think it's sending "0-" as the range string.

Comment (bdwheele) [reply lib/galaxy/web/framework/base.py:544]: I did a quick check on what's valid for the Range header and having an open-ended range is a legitimate request 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range

**Comment by mvdbeek** [comment]
Thanks for fixing this @bdwheele!

**Pull request merged**